governance.md

```diff
@@ -243,10 +264,164 @@ interactions with NumFOCUS.
+position by majority vote.
+
+
+### Release Manager(s)
+
+The Release Manager (RM) is appointed for a minor version (A.B.x)
+release series of Matplotlib.  They are responsible for the full
+release life cycle of all minor releases in the series including:
+
+- ensuring the whats new, API changes, and release notes are up to date
+- the timing of the releases
+- what changes should or should not be backported from the master
+  branch
+- rebuilding and publishing the website
+- announcing the release
+- publishing sdist and wheels to pypi
+- notifying down-stream packagers of the release
+
+An individual can be the RM as more than one release series at the
+same time.
+
+
+### Narrative Documentation Manager
+
+Matplotlib has a tremendous amount of documentation that is narrative
+in form.  This includes our examples and tutorials that live inside the
+main source repository and longer tutorials that live in other repositories
+in the Matplotlib organization.
+
+The Narrative Documentation Matplotlib (NDM) is responsible for
+shepherding all of this content including the scope, level, tone, and
+voice.
+
+### Reference Documentation Manager
+
+Matplotlib also has a tremendous amount of reference documentation
+embedded in doc strings.  This documentation needs to be complete and
+accurate as our users rely on it as the last authority of what a given
+method will do (short of reading the source).
+
+The Reference Documentation Manager (RDM) is responsible for ensuring that
+the docstrings are:
+
+- correctly formatted and render as intended
+- technically correct
+- complete
+
+### Community Manager
+
+The true strength of Matplotlib and why it has had such longevity as a
+project is the community of people around the code.  That community
+needs to be maintained.  The Community Manager (CM) is the is a
+catch-all for several very diverse tasks and this role may be split
+in the future and may want to enlist further assistants.
+
+**Communications**
+
+We need to maintain communication channels with the wider world.  To this end
+the CM is responsible for maintaining:
+
+- Matplotlib's social media presence
+- The Matplotlib blog
+- Recruiting people to give talks on Matplotlib at conferences / meetups
+- Maintaining a "matplotlib update" slide deck
+- the [Ann] mailing list
+
+This communication is primarily broadcast out and intended to engage
+with end-users.
+
+**Developer Relations**
+
+In addition to communication to end-users we need to maintain
+communication channels to developers of down-stream libraries and
+other expert users.
+
+- GSOC/mentoring programs
+- Recruiting people to run sprints
+- interaction with 3rd party extensions / publicity
+- [user]/[dev] mailing list moderation
+- Managing the discourse
+- recruiting new contributors
+
+**Event and Meeting planning**
+
+From time to time Matplotlib as an organization has in-person
+meetings.
+
+
+### API consistency Leader
+
```

Comment:
I'd consider merging this and reference documentation.

Reply:
I did this, but a private conversation with @timhoffm changed my mind back to it being two roles.

The API lead needs to have a broad view of the library and good design taste, the reference doc lead needs to be an expert at rst and documenting what the library actually does (independent if it is a good idea or not ;) )